pdns/rcpgenerator.cc

```diff
@@ -950,6 +982,25 @@ void RecordTextWriter::xfrSvcParamKeyVals(const set<SvcParam>& val) {
       d_string = str + '"' + d_string + '"';
       break;
     }
+    case SvcParam::ohttp: {
+      // no value
+      break;
+    }
+    case SvcParam::tls_supported_groups: {
+      auto str = d_string;
+      d_string.clear();
+      bool first = true;
+      for (auto const &group: param.getTLSSupportedGroups()) {
+        if (!first) {
+          str += ',';
+        }
+        str += std::to_string(group);
+        first = false;
+      }
+      d_string = str + d_string;
```

Comment:
Since you've `clear`ed `d_string` above, you can simply do
```suggestion
      d_string = str;
```

Reply:
done